01_fib/hsy3418/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(replaceSpaceWithlineBreak("1 1 2 3 5 8 13 "))
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func replaceSpaceWithlineBreak(line string) string {
+	return strings.Replace(line, " ", "\n", 7)
```

Comment:
If you change `7` to `-1`, then this function will work for all size strings (see https://golang.org/pkg/strings/#Replace)